packages/bodiless-organisms/src/components/Menu/SimpleMenu.tsx

```diff
@@ -13,6 +13,7 @@
  */
```

Comment:
For consistency, we should make the signature of withMenuDesign in MegaMenu match that in SimpleMenu. Either both should be overloaded to accept either a function or a design, or neither should be.